src/locales/en.json

```
@@ -21,5 +21,32 @@
   },
   "privacy": {
     "localStorage": "This website uses the browser <link /> to save user preferences, for example the test mode (regular or DSA) or the website language. These settings are never sent to any server, instead they are handled locally by the browser. Moreover, preferences are deleted after 6 months from the last change."
+  },
+  "results": {
+    "exitWarn": "You did not save your results. Are you sure to exit?",
+    "testPassed": "passed",
+    "testFailed": "failed",
+    "testPoints": "Score",
+    "tableHeader1": "Section score",
+    "tableHeader2": "Num. of questions",
+    "tableHeader3": "Correct",
+    "tableHeader4": "Wrong",
+    "tableHeader5": "Not given",
+    "pointsCalcTitle": "How the score is calculated",
+    "pointsCalcBody1": "The <strong>maximum achievable score</strong> <strong>is {{v1}} </strong> and is expressed to the second decimal place. <br /> The attribution of <strong>OFA TEST</strong> (Obblighi Formativi Aggiunti) occurs when the test score, rounded to the nearest integer, <strong>is less than {{v2}}</strong>.<br /> The attribution of <strong>OFA TENG</strong> occurs when, considering only the {{v3}} section, the number of correct answers <strong>is less than {{v4}}</strong>. <br />",
+    "pointsCalcUl1": "The test <strong>score</strong> is calculated by assigning:",
+    "pointsCalcItem1": "{{v}} point for each correct answer",
+    "pointsCalcItem2": "{{v}} point for each wrong answer",
+    "pointsCalcItem3": "{{v}} point for each answer not given",
```

It currently says "-0,25 point" and "0 point"

The lazy way to fix this would be changing the translation like this:
```suggestion
    "pointsCalcItem1": "{{v}} point for each correct answer",
    "pointsCalcItem2": "{{v}} points for each wrong answer",
    "pointsCalcItem3": "{{v}} points for each answer not given",
```

If we want to do it properly, we'd need a way to dynamically change between the singular and plural form, depedening on whether the value is `1`.
That would need to apply to the Italian version as well.